src/components/organisms/Accordion/AccordionHeader/AccordionHeader.test.tsx

```diff
@@ -20,7 +20,7 @@ describe('<AccordionHeader />', () => {
   it('renders the component with no a11y violations', async () => {
     const { container, getByTestId } = render(
       <Accordion aria-label="test-accordion">
-        <Accordion.Header data-automation="header" id="one" index={0} textSize="base">
+        <Accordion.Header data-automation="header" id="one" index={0} textSize="body">
```

Comment:
It looks like these haven't been updated since the API changed.

Reply:
Good catch!